cnf-certification-test/lifecycle/suite.go

```diff
@@ -448,6 +448,21 @@ func testPodsRecreation(env *provider.TestEnvironment) { //nolint:funlen
 		tnf.ClaimFilePrintf("%s", claimsLog.GetLogLines())
 		ginkgo.Fail("Some deployments or stateful sets are not in a good initial state. Cannot perform test.")
 	}
+
+	// Filter out pods with NodeSelectors present and FAIL them.
+	// We run into problems with this test when there are nodeSelectors affecting where
+	// pods are scheduled.  Also, they are not allowed in general, see the node-selector test case.
+	var podsWithNodeSelectors []*provider.Pod
+	for _, put := range env.Pods {
```

Comment:
`put` for Putin? `p` or `pod` is better.

Reply:
We use `put` everywhere for "Pod Under Test".